pages/playlist/curated/[id].vue

```diff
@@ -16,26 +16,33 @@ onBeforeMount(() => {
 </script>
 
 <template>
-  <div v-if="playlist">
-    <header class="flex gap-6 mb-12">
-      <ProtectedImage
-        v-if="playlist.cover"
-        :src="playlist.cover"
-        alt=""
-        class="aspect-square rounded-2xl bg-slate-100 w-[300px]"
-      />
-      <div class="p-6 flex flex-col justify-between">
-        <div>
-          <PageHeading>{{ playlist.title }}</PageHeading>
-          <p v-if="tracks">{{ tracks.length }} tracks</p>
+  <div class="grid grid-cols-12" v-if="playlist">
+    <div class="col-start-3 col-span-8">
+      <header class="flex gap-6 mb-12">
+        <ProtectedImage
+          v-if="playlist.cover"
+          :src="playlist.cover"
+          alt=""
+          class="aspect-square rounded-2xl bg-slate-100 w-[300px]"
+        />
+        <div class="p-6 flex flex-col justify-between">
+          <div>
+            <PageHeading>{{ playlist.title }}</PageHeading>
+            <p v-if="tracks">{{ tracks.length }} tracks</p>
+          </div>
+          <div class="flex gap-2">
+            <ButtonStyled intent="primary"> Shuffle </ButtonStyled>
+            <ButtonStyled>Follow</ButtonStyled>
+          </div>
         </div>
-        <div class="flex gap-2">
-          <ButtonStyled intent="primary"> Shuffle </ButtonStyled>
-          <ButtonStyled>Follow</ButtonStyled>
-        </div>
-      </div>
-    </header>
-    <TrackList :skeleton-count="10" :show-skeleton="pending" :tracks="tracks">
-    </TrackList>
+      </header>
+      <TrackList :skeleton-count="10" :show-skeleton="pending" :tracks="tracks">
+        <TrackItem
```

Comment:
It shouldn't be required to set `TrackItem` here, because `TrackList` doesn't have a `slot` tag anymore.